readme.txt

```diff
@@ -1,5 +1,5 @@
 === ACF Content Analysis for Yoast SEO ===
-Contributors: yoast, angrycreative, kraftner, marcusforsberg, viktorfroberg, joostdevalk, atimmer, jipmoors, theorboman
+Contributors: yoast, angrycreative, kraftner, marcusforsberg, viktorfroberg, joostdevalk, atimmer, jipmoors, theorboman, passoniate
```

Comment:
Could you please remove this? We appreciate your pull request but this is too much honor ;-)